programmers/서울에서-김서방-찾기.js

```diff
@@ -0,0 +1,9 @@
+function solution(seoul) {
+  var answer = "";
+  for (let i in seoul) {
+    if (seoul[i] == "Kim") {
+      answer = `김서방은 ${seoul.indexOf(seoul[i])}에 있다`;
```

Comment:
```suggestion
      answer = `김서방은 ${i}에 있다`;
```
사용하신 for in 반복문은 객체의 key를 반복해서 작업을 수행해요.
javascript에서 배열은 index를 key로, 각 원소들을 value로 하는 객체 형태입니다.

때문에 indexOf(seoul[i]) 보다 그냥 i만 사용해주시는게 좋을거 같아요

for of에 대해 혹시 모르신다면 이것도 한번 보시면 좋을거 같네요
[for of](https://jsdev.kr/t/for-in-vs-for-of/2938)